ZenPacks/zenoss/Docker/tests/test_data.py

```diff
@@ -234,6 +234,8 @@ def comparable_from_actual(actual):
         for k, v in actual.__dict__.items():
             if k == "maps":
                 comparable[k] = comparable_from_actual(v)
+            elif k in ("plugin_name"):
```

Comment:
Here is not tuple, but string. So it tests is `k` is substring of `"plugin_name"`. I'd rather use simple `elif k == "plugin_name":` here.